browser/ui/views/toolbar/brave_toolbar_view.h

```diff
@@ -69,6 +71,8 @@ class BraveToolbarView : public ToolbarView,
 
   BooleanPrefMember show_bookmarks_button_;
 
+  BooleanPrefMember show_side_panel_button_;
+
```

Comment:
just put them all together. no need to add empty lines.

Reply:
:+1: